docs/flush-tables-with-read-lock.md

```diff
@@ -1,5 +1,8 @@
 <!---
  review for rewrite and update
+
```

Comment:
Did you merge this comment as well? :smiley:

Reply:
I removed it